vmware/ngc/NGC-Register/src/main/resources/static/css/bootstrapValidator.css

```diff
@@ -0,0 +1,30 @@
+/**
```

Comment:
cannot use this validator externally without adding the code in our github? Is it ok to use? MIT should be ok...however please check...

Reply:
Yes it is ok to use. It is used internally.